src/Components/NavBar/NavBar.tsx

```diff
@@ -82,6 +84,10 @@ export const NavBar: React.FC = track(
   const isLoggedIn = Boolean(user)
   const showNotificationCount = isLoggedIn && !showMobileMenu
 
+  const isSearchDropDownImprovementsEnabled = useFeatureFlag(
+    "fx-force-search-dropdown-improvements"
+  )
+
```

Comment:
I am not sure why it's not getting enabled. Any idea, please?

Reply:
It's enabled for me. I also updated the strategies and the list of user ids (enabled ff only for you and me) in Unleash. Could you please check?

<img width="1627" alt="for-sam" src="https://user-images.githubusercontent.com/3513494/232754216-3cde40c9-90c9-4746-803c-a98bb91d8da3.png">

Reply:
Its still the same 🤔

![image](https://user-images.githubusercontent.com/17421923/232758267-6e5f6e00-e852-473a-a91e-0c9606395e8a.png)

Reply:
> [Force] Warning: cannot find flagName in featureFlags: 
do you see this warning in your console?